src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -374,12 +389,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <PropertyGroup Condition="'$(_DebugSymbolsProduced)' == 'true' and '$(OutputType)' == 'winmdobj'">
     <WinMDExpOutputPdb Condition="'$(WinMDExpOutputPdb)' == ''">$(IntermediateOutputPath)$(TargetName).pdb</WinMDExpOutputPdb>
-    <_WinMDDebugSymbolsOutputPath>$([System.IO.Path]::Combine('$(OutDir)', $([System.IO.Path]::GetFileName('$(WinMDExpOutputPdb)'))))</_WinMDDebugSymbolsOutputPath>
+    <_WinMDDebugSymbolsOutputPath>$(OutDir)$([System.IO.Path]::GetFileName('$(WinMDExpOutputPdb)'))</_WinMDDebugSymbolsOutputPath>
```

Comment:
I don't think I prefer the concatenation to the `Combine` call, even though it's generally safe. Do you think it's strongly preferable? If so, why?

Reply:
The only thing `Combine` does (_in this context_) is join the parts with a path separator which we can be sure, will be present in `OutDir` property.

So, if we want to be extra safe, sure I can add it. But in this case we shouldn't need to.

Also, Does using unnecessary property function calls impact perf?